CONTRIBUTING.md

```diff
@@ -22,6 +22,12 @@ Create a git branch using the convention of `github_user`/`issue_#` where `githu
 Create a test that replicates the problem/feature and fails and show how your fix results in a working test.
 Work on the code until your test, as well as all previous tests, pass.
 
+*Bonus points* -- ensure that [Sphinx](https://www.sphinx-doc.org/en/master/usage/restructuredtext/index.html) is still able to build the documentation in `docs`.
```

Comment:
is that a reasonable URL for sphinx? Why not just https://www.sphinx-doc.org/

Reply:
@speleo3 The top level of sphinx is pretty generic. I think we should only be interested in writing/supporting restructured text so the more specific URL seems more appropriate. Just my 2 cents.

Reply:
OK, thanks for explaining!